en/mulesoft/modules/ROOT/pages/ms_composer_prerequisites.adoc

```diff
@@ -297,23 +297,93 @@ To enable email notifications:
 . Toggle the button to enable email notifications.
 . Click *Save*.
 
+[[connect-composer-to-anypoint-platform]]
 == Connect Composer to Anypoint Platform
 
 Organizations that use both MuleSoft Composer and Anypoint Platform can connect the two products.
 
 To link Composer to Anypoint Platform:
 
-* Log in to Anypoint Platform.
-* In the navigation bar or the main Anypoint Platform page, click *Access Management*.
-* In the xref:access-management::managing-your-account.adoc[Access Management] navigation menu, click *Composer Sync*.
-* In the *Composer Sync Orgs* page, click *Add Composer Orgs*.
-* In the *Add organization* window, in the *Organization ID* box, enter your Composer organization and then click *Add*.
-+
-[NOTE]
+. In MuleSoft Composer, in the left pane, click *Settings* > *Account*. +
+The *Account* page appears.
 
-You can locate your Composer Organization ID by navigating to *Composer* > *Settings* > *Account* > *General Information*.
-+
-An email is generated and sent to the Composer organization admin that provides a link to sync Anypoint Platform to Composer.
+. On the *Account* page, copy the value in the *Organization ID* field.
+
+. Log in to the Anypoint Platform organization that you want to connect to MuleSoft Composer. +
+The *Anypoint Platform* page appears.
+
+. In the left pane, click xref:access-management::managing-your-account.adoc[*Access Management*]. +
+The *Access Management* page appears.
+
+. In the *Access Management* page, in the left pane, click *Composer Sync*. +
+The *Add Composer Organization* window appears.
+
+. In the *Add Composer Organization* window, in the *Organization ID* field, paste the *Organization ID* that you copied from the *Account* page of MuleSoft Composer and then click *Add*.
+
+. Open MuleSoft Composer and refresh the page.
+
+. In MuleSoft Composer, in the left pane, click *Settings* > *Account*. +
+The *Account* page appears, displaying a linking request from Anypoint Platform.
+
+. Click *Review and Confirm*.
+
+. In the *Access Token* section, click *Authenticate in Anypoint*. +
+The Anypoint Platform login page appears.
+
+. Log in to Anypoint Platform.
+
+. In the *Client App* section, click *Create Client App*. +
+The Composer organization is now linked to the Anypoint organization.
+
+== API Sharing
+
+MuleSoft Anypoint Platform customers can share externally available APIs managed in Anypoint Platform with business users for consumption in a MuleSoft Composer flow.
+
+Before sharing APIs from Anypoint Platform to MuleSoft Composer, ensure that:
+
+* Your organization uses Anypoint Platform for API Management.
+* Your organization has xref:exchange::to-create-an-asset.adoc#create-an-api-asset[Rest APIs] managed by API Manager that conform to the following:
+
+** API specifications use either RAML or OpenAPI
```

Comment:
```suggestion
** API specifications use either RAML or OpenAPI.
```

Reply:
Optional: I would also add "The" to the two list items: "The API specifications..." and "The API"...